bigbluebutton-html5/imports/ui/components/app/AppView.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Tabs from '../tabs/Tabs';
+import Header from '../header/Header';
+import Footer from '../footer/Footer';
+import Content from '../content/Content';
+import TabsPanel from '../tabs-panel/TabsPanel';
```

Comment:
Please create index for each of new components and import till folder name e.g. for Tabs use `../tabs` instead of `../tabs/Tabs`

Reply:
Done